spec/system/bitklavier_single_item_metadata_spec.rb

```diff
@@ -54,34 +54,34 @@
   end
 
   # rubocop:disable Layout/LineLength
-  xit "has expected citation information" do
-    visit '/catalog/78348'
-    apa_citation = "Stotler, D., F. Scotti, R.E. Bell, A. Diallo, B.P. LeBlanc, M. Podesta, A.L. Roquemore, & P.W. Ross. (2016). Midplane neutral density profiles in the National Spherical Torus Experiment [Data set]. Princeton Plasma Physics Laboratory, Princeton University."
-    expect(page).to have_content apa_citation
-    expect(page.html.include?('<button id="show-apa-citation-button"')).to be true
-    expect(page.html.include?('<button id="show-bibtex-citation-button"')).to be true
-  end
+  # xit "has expected citation information" do
```

Comment:
Please see my previous question regarding the removal for pending blocks with `xit`